python/foglamp/services/core/server.py

```diff
@@ -203,9 +205,11 @@ def _start_core(cls, loop=None):
             address, service_server_port = service_server.sockets[0].getsockname()
             _logger.info('Rest Server started on http://%s:%s', address, service_server_port)
 
-            cls.admin_announcer = ServiceAnnouncer('FogLAMP', '_foglamp._tcp', service_server_port, ['The FogLAMP Admin REST API'])
-            cls.user_announcer = ServiceAnnouncer('FogLAMP', '_foglamp_app._tcp', service_server_port,
-                                              ['The FogLAMP Application  REST API'])
+            cls.admin_announcer = ServiceAnnouncer('FogLAMP-{}'.format(unique_id), '_foglamp._tcp', service_server_port,
+                                                   ['The FogLAMP Admin REST API'])
+            # do we need this to announce as another service?
+            cls.user_announcer = ServiceAnnouncer('FogLAMP-App-{}'.format(unique_id), '_foglampapp._tcp', service_server_port,
```

Comment:
As above, we need a consistent name across reboots. We should use a configuration item to replace the FogLAMP part of the name.